Gemfile

```diff
@@ -22,7 +22,8 @@ group :test do
   gem "cucumber", "~> 2.1"
   gem "jekyll_test_plugin"
   gem "jekyll_test_plugin_malicious"
-  gem "nokogiri"
+  # nokogiri v1.8 does not work with ruby 2.1 and below
+  gem "nokogiri", RUBY_VERSION > "2.1" ? "~> 1.7" : "~> 1.7.0"
```

Comment:
Shouldn't it be something like:
`gem "nokogiri",  RUBY_VERSION < "2.1" ? "1.6" : "1.7"`